dataframes/03-from-pandas-to-dask.ipynb

```diff
@@ -0,0 +1,891 @@
+       "      <th>id</th>\n",
+       "      <th>name</th>\n",
+       "      <th>x</th>\n",
+       "      <th>y</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>timestamp</th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>2000-01-01 00:00:00</th>\n",
+       "      <td>1053</td>\n",
+       "      <td>Edith</td>\n",
+       "      <td>-0.082485</td>\n",
+       "      <td>0.126788</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2000-01-01 00:00:01</th>\n",
+       "      <td>992</td>\n",
+       "      <td>Zelda</td>\n",
+       "      <td>-0.292002</td>\n",
+       "      <td>0.814259</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2000-01-01 00:00:02</th>\n",
+       "      <td>991</td>\n",
+       "      <td>Yvonne</td>\n",
+       "      <td>-0.435019</td>\n",
+       "      <td>-0.923308</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2000-01-01 00:00:03</th>\n",
+       "      <td>998</td>\n",
+       "      <td>Xavier</td>\n",
+       "      <td>-0.164077</td>\n",
+       "      <td>0.195232</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2000-01-01 00:00:04</th>\n",
+       "      <td>989</td>\n",
+       "      <td>Norbert</td>\n",
+       "      <td>0.343440</td>\n",
+       "      <td>-0.787839</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                       id     name         x         y\n",
+       "timestamp                                             \n",
+       "2000-01-01 00:00:00  1053    Edith -0.082485  0.126788\n",
+       "2000-01-01 00:00:01   992    Zelda -0.292002  0.814259\n",
+       "2000-01-01 00:00:02   991   Yvonne -0.435019 -0.923308\n",
+       "2000-01-01 00:00:03   998   Xavier -0.164077  0.195232\n",
+       "2000-01-01 00:00:04   989  Norbert  0.343440 -0.787839"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "pdf = ddf.compute()  # create a pandas dataframe\n",
+    "print(type(pdf))\n",
+    "pdf.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have both `dataframes` we can start to compair the interactions with them"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Conceptual shift - from Update to Insert/Delete\n",
+    "Dask does not update - thus no arguments such as \"inplace= True\" which exist in Pandas.  \n",
+    "For more detials see [issue#653 on github](https://github.com/dask/dask/issues/653)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1.1 Rename"
```

Comment:
I would remove this one. I don't consider `.rename(..., inplace=True)` to be a best practice, and there has been proposals to deprecate inlace in many places in pandas.

I would recommend `df = df.rename(columns=...)`, which works for both pandas and dask.

Reply:
I think there is a value to show that if we use `inplace=True` we get an error